config-connector/solutions/iam/helm/pubsub-topic/README.md

```diff
@@ -0,0 +1,95 @@
+# Pub/Sub Topic
+
+==================================================
+
+## NAME
+
+  pubsub-topic
+
+## SYNOPSIS
+
+  
+  Config Connector compatible YAML files to create a Pub/Sub topic in your desired project, and grant a specific member a role (default to roles/pubsub.editor) for accessing the Pub/Sub topic that just created
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the service account folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/iam/helm/pubsub-topic
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](../../../README.md#helm)
+
+## USAGE
+
+All steps are run from the current directory ([config-connector/solutions/iam/helm/pubsub-topic](.)).
+
+1. Review and update the values in `./values.yaml`.
+
+1. Validate and install the sample with Helm.
+
+    ```bash
+    # validate your chart
+    helm lint . --set iamPolicyMember.iamMember=user:name@example.com
+
+    # check the output of your chart
+    helm template . --set iamPolicyMember.iamMember=user:name@example.com
+
+    # Do a dryrun on your chart and address issues if there are any
+    helm install . --dry-run --set iamPolicyMember.iamMember=user:name@example.com --generate-name
+
+    # install your chart
+    helm install . --set iamPolicyMember.iamMember=user:name@example.com --generate-name
+    ```
+
+1. _Optionaly_, you can customize optional values by explictly setting them when installing the solution:
```

Comment:
Could you clarify what the optional values are in the doc? It might confuse users.

Reply:
i changed it to 
1. _Optionaly_, you can set the name of the PubSub topic (defaults to `allowed-topic`) and the role to grant (defaults to `roles/pubsub.editor`, full list of roles [here](https://cloud.google.com/iam/docs/understanding-roles#pub-sub-roles)) by explictly setting them when installing the solution: